source/tests/system/visa_driver_api_tests.cpp

```diff
@@ -186,14 +189,54 @@ class VisaMessageBasedLoopbackTest : public VisaDriverApiTest {
     EXPECT_EQ(data.size(), response.return_count());
   }
 
+  void write_async(const std::string& data)
```

Comment:
Would it make sense for all of this to be in a single method (possibly a streaming one?) for `grpc-device`? Is there anything similar in a LV API (or other higher-level language)?

Matching the low-level driver API (like this) is a good default for `grpc-device`. We can always add higher-level/async variants later. So, I don't have a problem with it as-is.

Reply:
I agree this could make sense as a higher-level abstraction to add later. We need the 1:1 mapping for VISA to work properly in LabVIEW. I'm not going to mark this as an action item though, just pointing out that it's a reasonable feature someday.